WoloxAndroidBootstrp/app/src/main/java/ar/com/wolox/android/example/network/builder/NetworkBuilder.kt

```diff
@@ -1,27 +1,26 @@
 package ar.com.wolox.android.example.network.builder
 
 import ar.com.wolox.wolmo.networking.retrofit.handler.NetworkResponse
-import okhttp3.ResponseBody
 import retrofit2.Response
 
 class NetworkBuilder<T> {
 
     private var onResponseSuccessful: (T?) -> Unit = {}
-    private var onResponseFailed: (ResponseBody?, Int) -> Unit = { _, _ -> }
```

Comment:
You should never change this file at all. If the endpoint returns an error, it's not okay to map it to your success structure (in this case, LoginResponse). Check the endpoint's response and handle that `message` parameter :)